common/sgx/internal.edl

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/*
+**==============================================================================
+**
+** internal.edl:
+**
+**     This file declares internal ECALLs/OCALLs used by liboehost/liboecore.
+**
+**==============================================================================
+*/
+
+enclave {
+
+    trusted {
+
+        /* This function returns oe_internal_ping_ocall(value). */
+        public int oe_internal_ping_ecall(int value);
```

Comment:
Why is this EDL file under common/sgx?  It doesn't look SGX specific.

Reply:
This infrastructure is replacing all the SGX-specific **oe_ecall** and **oe_ocall** function calls, so it is in the sgx directory.